lib/index.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @author Toru Nagashima <https://github.com/mysticatea>
+ * See LICENSE file in root directory for full license.
+ */
+"use strict";
+
+const KEYS = require("./visitor-keys.json");
+
+// Types.
+const NODE_TYPES = Object.freeze(Object.keys(KEYS));
```

Comment:
Does `NODE_TYPES` need to be frozen? It seems like it's not exported.

(I suppose it's not a problem to freeze it either way.)

Reply:
Because of my liking. It clarifies that it's a constant.